<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="40.134" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2024-07-29T15:19:29" tests="0" file="cypress\e2e\place_order_login_before_checkout.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Place Order: Register Before Checkout" timestamp="2024-07-29T15:19:30" tests="1" time="39.715" failures="1">
    <testcase name="Place Order: Register Before Checkout &quot;before all&quot; hook: Registration before login to the system and purchase product for &quot;Register before cart adds products into the cart, and process shippment&quot;" time="0.000" classname="&quot;before all&quot; hook: Registration before login to the system and purchase product for &quot;Register before cart adds products into the cart, and process shippment&quot;">
      <failure message="`cy.visit()` failed trying to load:

http://automationexercise.com/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  &gt; Error: ESOCKETTIMEDOUT

Common situations why this would fail:
  - you don&apos;t have internet access
  - you forgot to run / boot your web server
  - your web server isn&apos;t accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Place Order: Register Befor...`" type="CypressError"><![CDATA[CypressError: `cy.visit()` failed trying to load:

http://automationexercise.com/

We attempted to make an http request to this URL but the request failed without a response.

We received this error at the network level:

  > Error: ESOCKETTIMEDOUT

Common situations why this would fail:
  - you don't have internet access
  - you forgot to run / boot your web server
  - your web server isn't accessible
  - you have weird network configuration settings on your computer

Because this error occurred during a `before all` hook we are skipping the remaining tests in the current suite: `Place Order: Register Befor...`
    at <unknown> (http://localhost:3347/__cypress/runner/cypress_runner.js:131480:74)
    at visitFailedByErr (http://localhost:3347/__cypress/runner/cypress_runner.js:130878:12)
    at <unknown> (http://localhost:3347/__cypress/runner/cypress_runner.js:131479:11)
    at tryCatcher (http://localhost:3347/__cypress/runner/cypress_runner.js:1807:23)
    at Promise._settlePromiseFromHandler (http://localhost:3347/__cypress/runner/cypress_runner.js:1519:31)
    at Promise._settlePromise (http://localhost:3347/__cypress/runner/cypress_runner.js:1576:18)
    at Promise._settlePromise0 (http://localhost:3347/__cypress/runner/cypress_runner.js:1621:10)
    at Promise._settlePromises (http://localhost:3347/__cypress/runner/cypress_runner.js:1697:18)
    at _drainQueueStep (http://localhost:3347/__cypress/runner/cypress_runner.js:2407:12)
    at _drainQueue (http://localhost:3347/__cypress/runner/cypress_runner.js:2400:9)
    at Async._drainQueues (http://localhost:3347/__cypress/runner/cypress_runner.js:2416:5)
    at Async.drainQueues (http://localhost:3347/__cypress/runner/cypress_runner.js:2286:14)
From Your Spec Code:
    at Context.eval (webpack:///./cypress/e2e/place_order_login_before_checkout.cy.js:4:11)

From Node.js Internals:
  Error: ESOCKETTIMEDOUT
      at ClientRequest.<anonymous> (<embedded>:2049:106060)
      at Object.onceWrapper (node:events:628:28)
      at ClientRequest.emit (node:events:514:28)
      at TLSSocket.emitRequestTimeout (node:_http_client:847:9)
      at Object.onceWrapper (node:events:628:28)
      at TLSSocket.emit (node:events:526:35)
      at TLSSocket._onTimeout (node:net:571:8)
      at listOnTimeout (node:internal/timers:569:17)
      at process.processTimers (node:internal/timers:512:7)]]></failure>
    </testcase>
  </testsuite>
</testsuites>